src/App.js

```diff
@@ -1,25 +1,44 @@
-import logo from './logo.svg';
+
 import './App.css';
+import contactsArr from "./contacts.json";
+import {useState} from "react";
+
+
 
 function App() {
-  return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
-}
+
+  let fiveContacts = contactsArr.splice(0, 5)
+
+  const [contacts, setContacts] = useState(fiveContacts)
+  
+      return(
+        <div className="App">
+        <table>
+          <tr>
+            <th>Picture</th>
+            <th>Name</th>
+            <th>Popularity</th>
+            <th>Won an Oscar</th>
+            <th>Won an Emmy</th>
+          </tr>
+
+          {contacts.map((props) => {
+            return (
+            <tr>
+            <td className="pictureUrl"> 
+              <img src={props.pictureUrl}></img>
+            </td>
+            <td className="name">{props.name}</td>
+            <td className="popularity">{props.popularity}</td>
+          </tr>
+          )
+          })}
+          
+        </table>
+        </div>
+      )
```

Comment:
If I was looking for somethign to pick on ;)  Your table should have <thead> tag  -> for headers and <tbody> tag for content . Like the structure below

```
<table>
   <thead>
       <tr>
            <th>Picture</th>
            <th>Name</th>
       </tr>
   </thead>
   <tbody>
       {contacts.map(() => {
            return (
                  <tr>
                      <td>Actor's picture here</td>
                     <td>Actor's name here</td>
                  </tr>
            )
       )}
   </tbody>
</table>
```